Source/WebKit/UIProcess/Cocoa/WKShareSheet.mm

```diff
@@ -336,11 +336,10 @@ - (void)presentWithShareDataArray:(NSArray *)sharingItems inRect:(std::optional<
 #endif // PLATFORM(VISION)
     {
         UIPopoverPresentationController *popoverController = [_shareSheetViewController popoverPresentationController];
-        if (rect) {
-            popoverController.sourceView = webView;
-            popoverController.sourceRect = *rect;
-        } else
-            popoverController._centersPopoverIfSourceViewNotSet = YES;
+        popoverController.sourceView = webView;
+        popoverController.sourceRect = rect.value_or(webView.bounds);
+        if (!rect)
+            popoverController.permittedArrowDirections = UIPopoverArrowDirectionNone;
```

Comment:
Oh, as it turns out, `UIPopoverArrowDirectionNone` is more SPI 🤦🏻‍♂️

It seems I'll need to swap this out for just `0`.